k8s/namespaces/rd/rd-commondata-dataload/rd-commondata-api.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
```

Comment:
is file name correct ? since this is data load api ?

Reply:
renamed file name to rd-commondata-dataload